sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/Vendoring/OpenTelemetry.Instrumentation.SqlClient/Implementation/SqlActivitySourceHelper.cs

```diff
@@ -1,18 +1,7 @@
 // <copyright file="SqlActivitySourceHelper.cs" company="OpenTelemetry Authors">
```

Comment:
Where is this line coming from? It is not in the original file https://github.com/open-telemetry/opentelemetry-dotnet/blob/bae8a6b6991b5072c1b6d8412e329257cdfca953/src/OpenTelemetry.Instrumentation.SqlClient/Implementation/SqlActivitySourceHelper.cs#L1-L2

Reply:
We got advice from CELA last year about how to copy code from an open source repo.
I can't find the email thread right now, but I'll try to find and share.

This line was added to all the vendored files in our original PR. https://github.com/Azure/azure-sdk-for-net/pull/38848
In THIS PR, I'm trying to bring these other files in line with that original PR.

Reply:
I don't understand.

Reply:
CORRECTION
This line was originally included in the files from the OTel repo. 
Reiley removed them in the source files a few months ago. https://github.com/open-telemetry/opentelemetry-dotnet/pull/5140
We should do the same here. I'll create another PR to get this cleaned up.

Reply:
Now coming to my next question - how do we pull 3rd party code and how do we review these PRs? (it's hard to tell whether the files are the same or there are accidental mistakes during copy/paste).

Reply:
Our approach should be to copy the entire file, not to compare the text and merge them. I spoke to @TimothyMothra, stating that we need to copy the files instead of comparing the text before the review. I also missed this during the review.

We don’t have a formalized process for review. We will work with the Azure SDK team on the process. Until we find a process, we may need to have a very strict manual review process. (Along with it, we should at least have minimum of 2 approval needed to merge).

Reply:
> Our approach should be to copy the entire file, not to compare the text and merge them. I spoke to @TimothyMothra, stating that we need to copy the files instead of comparing the text before the review. I also missed this during the review.
> 
> We don’t have a formalized process for review. We will work with the Azure SDK team on the process. Until we find a process, we may need to have a very strict manual review process. (Along with it, we should at least have minimum of 2 approval needed to merge).

Thanks!